Apache2 conf

devendermishrajio opened the pull request:
Installation of apache2 package and configuration.

Commit:
Add apache2 conf

Commit:
Remove apache2 as it is already getting installed

Commit:
Add SSL and sites

Commit:
Remove jiocloud_ssl_certificate

1. Remove jiocloud_ssl certificate package. It will come from command line.
2. ServerName from variable.
3. SSL Certificates are variable now.

Commit:
Set servername to endpoint

Commit:
Make servername variable provided by the user

Commit:
Use certificate name as user parameter

akash1808 commented on controller.yml:
```diff
@@ -44,3 +44,26 @@
      - apache2
   - include: controller_extension.yml
+
+- hosts: cc*
```
would it be possible to separate out this file? It would be great as it will help in applying atomic scripts.

akash1808 commented on controller_apache2.yml:
```diff
@@ -0,0 +1,49 @@
+    - headers
+    - ssl
+
+- name: Install jiocloud ssl certs
```
Separate the installation of jiocloud_ssl_certs and all the changes related to it. This will happen in reusability of same.

akash1808 commented on controller_apache2.yml:
```diff
@@ -0,0 +1,49 @@
+- name: Refresh CA list
+  command: update-ca-certificates --fresh
+
+- name: Remove sites-available
```
No need to remove complete folder. This will lead to deletion of files every time  playbooks run. Further,   ansible can just apply diff changes in vhost file. We can delete the default file if required.

devendermishrajio replied:
Deletion in sites-enabled is retained as it is link here. If not deleted, it will interfere in applying next time by giving error.

akash1808 commented on files/apache2-site.conf:
```diff
@@ -0,0 +1,54 @@
+# ************************************
```
Also, please keep templates in templates folder instead of files folder.

Commit:
Changes as per review

1. templates are moved to templates folder
2. Jiocloud SSL is moved to separate file.
3. Apache2 installation and configuration is separate from
controller.yml.

devendermishrajio commented:
Changes are made. Please review them.

varunarya10 commented on controller_apache2_conf.yml:
```diff
@@ -0,0 +1,48 @@
+      site_proxyport: 19292
+    - name: glance-registry
+      site_port: 9191
+      site_admin: cloud.devops@ril.com
```
please replace with JioCloud.ComputeTeam@ril.com
